src/components/picker/examples/picker-multiple.tsx

```diff
@@ -2,7 +2,7 @@ import { ListItem } from '@limetech/lime-elements';
 import { Component, h, State } from '@stencil/core';
 
 /**
- * Multiple values
+ * Multiple values can be picked
```

Comment:
You think the change in the header looks better?

Reply:
Yes, I think the new header explains the example more clearly.